enderio-machines/src/main/java/com/enderio/machines/client/gui/screen/WirelessChargerScreen.java

```diff
@@ -0,0 +1,84 @@
+package com.enderio.machines.client.gui.screen;
+
+import com.enderio.base.api.EnderIO;
+import com.enderio.base.client.gui.widget.EIOCommonWidgets;
+import com.enderio.base.client.gui.widget.RedstoneControlPickerWidget;
+import com.enderio.base.common.lang.EIOLang;
+import com.enderio.machines.client.gui.screen.base.MachineScreen;
+import com.enderio.machines.client.gui.widget.ActivityWidget;
+import com.enderio.machines.client.gui.widget.CapacitorEnergyWidget;
+import com.enderio.machines.common.blocks.obelisks.aversion.AversionObeliskMenu;
+import com.enderio.machines.common.blocks.wireless_charger.WirelessChargerMenu;
+import net.minecraft.client.gui.GuiGraphics;
+import net.minecraft.network.chat.Component;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.world.entity.player.Inventory;
+
+public class WirelessChargerScreen extends MachineScreen<WirelessChargerMenu> {
+
+    private static final ResourceLocation BG_TEXTURE = EnderIO.loc("textures/gui/screen/wireless_charger.png");
+    private static final int WIDTH = 176;
+    private static final int HEIGHT = 166;
+
+    public WirelessChargerScreen(WirelessChargerMenu pMenu, Inventory pPlayerInventory, Component pTitle) {
+        super(pMenu, pPlayerInventory, pTitle);
+        imageWidth = WIDTH;
```

Comment:
I think we have a different method to do this

Reply:
Sorry, no sure what you are referring to here.

Reply:
Seems I was confused by the old design